client/jetpack-cloud/sections/partner-portal/lib/get-product-title.ts

```diff
@@ -5,8 +5,8 @@
  * @returns Product title
  */
 export default function getProductTitle( product: string, removeVariant: boolean = false ): string {
```

Comment:
This removes the brand part and the variant part (10GB or 1TB). The title matches the titles on jetpack.com/pricing